telosnetwork/telos-wallet
# Fetch balances and send transactions (WalletConnect)

# Fixes #293 #277

## Description
see issues

## Test scenarios

Mobile
1. Login using WalletConnect
2. Should display token balances
3. Click 'send', select system token (TLOS) and complete a transaction
4. Should automatically open selected wallet and return to app on confirmation 
5. repeat steps 3 & 4 for a non-system token (STLOS, USDT, etc) 

Desktop
1. MetaMask login (including via WalletConnect) should display balances and allow completion of transactions

Notes: 
- Known issues: #296 

## Checklist:
